uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/leave-request-instance_test.js

```diff
@@ -206,6 +214,7 @@ define([
         LeaveRequestAPI.update.and.returnValue(defer.promise);
         defer.resolve(jasmine.any(Object));
         spyOn(LeaveRequestInstance, 'toAPI').and.returnValue(toAPIReturnValue);
+        LeaveRequestInstance.comments = commentsData.getCommentsWithMixedIDs().values;
```

Comment:
I have reused existing `LeaveRequestInstance`. Also `LeaveRequestInstance` is created from dependency injection and upper case is used for all similar cases.